src/package-json/package-json.js

```diff
@@ -67,11 +85,12 @@ export default class PackageJson {
     return JSON.stringify(this.toPlainObject(), null, 4);
   }
 
-  async write({ override = true }: { override?: boolean }): Promise<boolean> {
+  async write({ override = true , postInstallLinkData = [] }: { override?: boolean, postInstallLinkData: Object }): Promise<boolean> {
```

Comment:
the postInstallLinkData type is Object but the default is []... one of them should be changed